CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # CHANGELOG
 
-## 0.21.1dev
```

Comment:
@edblancas what's the approach here, are we keeping the `dev` version for dev string updates?

Reply:
yeah the dev stays until we make a new release, once I run the release process 0.21.1dev becomes 0.21.1 (or 0.22 in case where making a major release)